src/main/java/org/radarcns/mongo/util/MongoHelper.java

```diff
@@ -100,17 +101,18 @@ public String getParam() {
             String source, Long start, Long end, MongoCollection<Document> collection) {
         FindIterable<Document> result = collection.find(
                 Filters.and(
-                        eq(USER, subject),
```

Comment:
the filter probably needs to include the `KEY/PROJECT_ID`, `KEY/USER_ID`, etc.

Reply:
I am working on a separate PR to address new collection structure